src/components/material/StockAndReservationInfo.tsx

```diff
@@ -4,7 +4,7 @@ import { useText } from "../../core/utils/text";
 
 export interface StockAndReservationInfoProps {
   stockCount: number;
-  reservationCount: number;
+  reservationCount?: number;
```

Comment:
Why not remove it totally?
Is it needed elsewhere?
Would be nice to have that kind of information in the commit, so it is possible to follow your thoughts

Reply:
Yes, it is needed elsewhere :)